grafana/deployment.yaml

```diff
@@ -284,6 +296,19 @@ data:
           authenticationType: gce
           defaultProject: $GCP_PROJECT
         editable: true
+{% elif global.cloud == "azure" %}
+      - name: Azure Monitor
+        type: grafana-azure-monitor-datasource
+        access: proxy
+        jsonData:
+          azureAuthType: clientsecret
+          cloudName: azuremonitor
+          tenantId: d6c9f2ea-d3bb-4ca9-8b14-231bac999aa6
+          clientId: 130c8cb9-4175-430e-919d-9d74282f2ffa
+          subscriptionId: 22cd45fe-f996-4c51-af67-ef329d977519
```

Comment:
Can we not hard-code these? You should be able to get `{{ global.azure_subscription_id }}` in CI for example, and the tenant and client IDs should be in the gsa-key secret.